package.json

```diff
@@ -36,7 +36,7 @@
     "test:nyc:lcov": "nyc report --reporter=text-lcov > .lcov.info",
     "test:nyc:report": "nyc report",
     "test:repoutils": "yarn workspace botbuilder-repo-utils test",
-    "test:schemas": "mocha \"libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js\" --exit",
+    "test:schemas": "mocha \"libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js\" --exit --bail",
```

Comment:
Note: Doesn't make sense to run the second test if the first fails.

Reply:
Do you need the inner escaped quotes?

Reply:
Not sure. I was following [existing patterns](https://github.com/microsoft/botbuilder-js/blob/main/package.json#L31)

Reply:
Removed. Doesn't seem to make a difference